pandas/tests/io/test_sql.py

```diff
@@ -146,13 +146,17 @@ def create_and_load_iris_sqlite3(conn: sqlite3.Connection, iris_file: Path):
         reader = csv.reader(csvfile)
         next(reader)
         stmt = "INSERT INTO iris VALUES(?, ?, ?, ?, ?)"
-        cur.executemany(stmt, reader)
+        cur.executemany(stmt, list(reader))
+
+    conn.commit()
+    cur.close()
```

Comment:
Those changes here are not strictly required now, since you reused the `create_and_load_iris` function below, and not this one